CONTRIBUTING.md

```diff
@@ -87,21 +87,16 @@ You can run the entire test suite using `pnpm test`. If you wish to only run e2e
 
 By default, `pnpm test:int` will only run int test against MongoDB. To run int tests against postgres, you can use `pnpm test:int:postgres`. You will have to have postgres installed on your system for this to work.
 
-### Commits
+### Pull Request Titles
 
-We use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for our commit messages. Please follow this format when creating commits. Here are some examples:
+We squash PR commits and use the PR title as the commit message. For that reason, please use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for your PR titles. Here are some examples:
 
-- `feat: adds new feature`
-- `fix: fixes bug`
-- `docs: adds documentation`
-- `chore: does chore`
-
-Here's a breakdown of the format. At the top-level, we use the following types to categorize our commits:
-
-- `feat`: new feature that adds functionality. These are automatically added to the changelog when creating new releases.
-- `fix`: a fix to an existing feature. These are automatically added to the changelog when creating new releases.
-- `docs`: changes to [docs](./docs) only. These do not appear in the changelog.
-- `chore`: changes to code that is neither a fix nor a feature (e.g. refactoring, adding tests, etc.). These do not appear in the changelog.
+- `feat: add new feature`
+- `fix: fix bug`
+- `docs: add documentation`
+- `test: add/fix tests`
+- `refactor: refactor code`
+- `chore: anything that does not fit into the above categories`
 
 If you are committing to [templates](./templates) or [examples](./examples), use the `chore` type with the proper scope, like this:
```

Comment:
We should elaborate more on "scopes". We mention this below, but we should be more descriptive about packages within the monorepo and provide examples. For example, `chore(next): this is a chore`. The listed examples are all "unscoped", i.e. Payload core.